.github/actions/setup/action.yml

```diff
@@ -0,0 +1,13 @@
+name: Set up pandas
```

Comment:
can this be in more of the builds? (you change a few i c); normally do this as a separate PR (e.g. when you refactor want to do one thing), but ok to leave.

Reply:
> can this be in more of the builds?

Maybe! The environment gets set up various ways for various builds. I wasn't sure why, so just changed the ones that looked the same.

> normally do this as a separate PR

The refactoring was in service of splitting the two jobs, but noted, thanks.